helm/lisk-core/values.yaml

```diff
@@ -0,0 +1,39 @@
+---
+replicaCount: 1
+
+persistence:
+  enable: true
+  storage: 10Gi
```

Comment:
We should add a comment explaining this is used for postgrtes and can be expanded.